lib/dl_api_lib/dl_api_lib/connector_availability/base.py

```diff
@@ -76,6 +76,8 @@ def as_dict(self, localizer: Localizer) -> dict[str, Any]:
 
 @attr.s(kw_only=True)
 class ConnectorBase(LocalizedSerializable, metaclass=abc.ABCMeta):
+    visibility_mode: ConnectorAvailability = None
```

Comment:
For the sake of consistency, let's make is a prop, just like `hidden` or `conn_type_str`